products/BellHybrid/apps/common/src/AlarmModel.cpp

```diff
@@ -110,13 +110,13 @@ namespace app
         }
         return alarmEventPtr->enabled;
     }
-    std::uint32_t AlarmModel::getSnoozeDuration()
+    std::chrono::seconds AlarmModel::getSnoozeDuration() const
```

Comment:
Shouldn't std::chrono::minutes be the return value here?

Reply:
Actually, it was the source of the bug. Snooze duration is stored using minutes but layers that use this method require std::time_t which is in seconds.